le-taxi-api-node.js/src/databaseMigrations/postgres/afterSemver/pg_1_1_1_add_vehicle_property_bonjour.ts

```diff
@@ -0,0 +1,8 @@
+// Licensed under the AGPL-3.0 license.
+// See LICENSE file in the project root for full license information.
+import { postgrePool } from "../../../features/shared/taxiPostgre/taxiPostgre";
+
+export async function pg_1_1_1_add_vehicle_property_bonjour(): Promise<void> {
+  await postgrePool.query(`ALTER TABLE public."vehicle_description"
+  ADD COLUMN IF NOT EXISTS bonjour boolean DEFAULT false;`);
```

Comment:
Majority of boolean column are tri-state (true/false/null) [see here](https://github.com/VilledeMontreal/montreal-taxi-registry/blob/9de6dc656b1604e54878c6b1a9c17f6d3bb0c584/le-taxi-api-node.js/src/databaseMigrations/postgres/beforeSemver/initial/txp.sql#L487-L510)
Should we uniformize and have tri-state also for bonjour ?

Reply:
It is a tri state, in SQL columns are NULLable by default. You need to be explicit about the NOT NULL constraint to only allow for true/false values =>
https://www.postgresql.org/docs/current/ddl-default.html#DDL-DEFAULT
https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-NOT-NULL